app/assets/javascripts/short-form/ShortFormNavigationService.js.coffee

```diff
@@ -111,6 +111,11 @@ ShortFormNavigationService = ($state) ->
           'household-members'
         else
           'household-intro'
+      when 'general-lottery-notice'
+        'status-programs'
+      when 'status-vouchers'
```

Comment:
I simplified the back state for this. ideally id be able to call $scope.checkIfPreferencesApply (couldn't coz couldn't inter-inject dependency) and determine route from there, but routing to status-programs is a catch-all. open to suggestions!